app/models/article.rb

```diff
@@ -416,6 +416,17 @@ def access_by?(user)
     user.admin? || user_id == user.id
   end
 
+  def merge_with(other_article_id)
+    # keep title and author from article 1 (self)
+    article_2 = Article.find(other_article_id.to_i)
```

Comment:
Even though you are merging with another article, it doesn't really make sense to call this variable `article_2` since you use the `self` variable for the "article_1".